scripts/populate_preprint_providers.py

```diff
@@ -60,7 +59,13 @@ def set_or_update_m2m_fields(provider, emails, licenses, links, social_accounts)
         provider.licenses_acceptable.add(*licenses)
 
     for link in links:
-        obj, created = PreprintProviderLink.objects.get_or_create(**link)
+        obj, created = provider.links.update_or_create(
```

Comment:
https://docs.djangoproject.com/en/1.11/ref/models/querysets/#update-or-create
> The update_or_create method tries to fetch an object from database based on the given kwargs. If a match is found, it updates the fields passed in the defaults dictionary.

The situation you described in your first comment is the desired behavior here (and based on my interpretation of the django docs, that's what will happen).